src/components/Carousel.js

```diff
@@ -203,9 +204,15 @@ class Carousel extends Component {
     }
 
     stopOnHover = () => {
+        this.setState({isMouseEntered: true});
```

Comment:
@leandrowd  I've now added tests for the isMouseEntered state change and the stop/start auto play functionality if a mouse enters/leaves.